packages/layouts-css/src/page.scss

```
@@ -95,4 +107,69 @@
       }
     }
   }
+
+  &-bottom-bar {
+    grid-area: bottom-bar;
+    height: calc(var(--iui-size-s)*5 + var(--iui-size-l)); //this is the sizes of the two bottom buttons of side nav
+
+    @include mobile {
+      height: fit-content;
+    }
+  }
+
+  &-action-bar {
+    background-color: var(--iui-color-background);
+    border-top: 2px solid var(--iui-color-border);
+    margin-inline: auto;
+    padding: var(--iui-size-m);
```

Padding on bigger screens seems to be smaller than in mid size screens. Maybe this could work.

```suggestion
    padding-inline: unquote('max(calc(var(--iui-size-xl) * 2), env(safe-area-inset-right))') unquote('max(calc(var(--iui-size-xl) * 2), env(safe-area-inset-left))');

```